src/rprblender/export/mesh.py

```diff
@@ -112,6 +113,38 @@ def init_from_mesh(mesh: bpy.types.Mesh, calc_area=False, obj=None):
 
         if calc_area:
             data.area = sum(tri.area for tri in mesh.loop_triangles)
+           
+        # Not possible to store strings/ bools in primvars. If the type was bool, it would be possible to just pass 1 or 0, and let it be interpolated, and then round it, but that would require extra code in the shaderattribute node
+        # Also cannot currently handle anything besides vertex attributes (point/corner), without triangulating the mesh first, which is over my head.
```

Comment:
I guess I'm not clear here.  Why would you need to triangulate a mesh?  Blender gives you attribues in what interpolation?  Vertex / Uniform / Constant?  

Anything thats varying across vertices is handled at the renderer to interpolate.

Reply:
In blender if you have a cube, and create an attribute with the domain of "face" you will get six values across the entire cube. The mesh is triangulated before it is sent to prorender, and I'm not sure how to get triangle level attributes out of blender *after* triangulation occurs. if you try to submit the six values to prorender as primvars, it throws a parameter error because the total values don't match the mesh triangle count